nix/static-binary.sh

```diff
@@ -83,5 +84,5 @@ tar -C "${result}" \
   --exclude share/vast/test \
   --exclude share/vast/integration \
   --mode='u+w' \
-  -cvzf "$PWD/build/${target}-${desc}-static.tar.gz" \
```

Comment:
Is the `-linux` suffix for consistency with the other CI artifacts?

Reply:
Exactly.